assignment2.txt

```diff
@@ -0,0 +1,11 @@
+# Julia Tan
+
+1. Depends. Two or more operators can be combined in the same line of code if they are arithmetic operators (+, -, *, /). When using the greater, less than, or equals operator, it cannot be used more than once in the same statement, unless they are separated with || or &&.
+2. A logical error creates an unexpected outcome, while a syntax error is only an error in grammar.
```

Comment:
Correct - it's important to note that logical errors occur at runtime whereas syntax errors occur at compile time.